lib/app/features/feed/stories/views/components/story_viewer/components/core/story_input_field.dart

```diff
@@ -18,10 +20,23 @@ class StoryInputField extends HookWidget {
   final double bottomPadding;
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     final isTextNotEmpty = useState(false);
     final focusNode = useFocusNode();
 
+    useEffect(
```

Comment:
`useNodeFocused` would you mind using this hook?

Reply:
It will not work because useNodeFocused returns ValueListenable. In code, it will look like this:

 ```
 final focusNode = useFocusNode();
    final focused = useNodeFocused(focusNode);

    useEffect(
      () {
        ref.read(storyPauseControllerProvider.notifier).paused = focused.value;
        return null;
      },
      [focused.value],
    );
```
Even I replace `useEffect` with `useOnInit` I get the not expected behaviour